src/main/java/frc/robot/subsystems/PneumaticHubSubsystem.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.subsystems;
```

Comment:
do we need this subsystem?

Reply:
yes
it allows us to declaratively create solenoids from a specific module
and without a subsystem, this would cause resource allocation errors